firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalStoreTestCase.java

```diff
@@ -389,7 +389,8 @@ public void testHandlesSetMutationThenDocument() {
             asList(targetId),
             emptyList()));
     assertChanged(doc("foo/bar", 2, map("foo", "bar")).setHasLocalMutations());
-    assertContains(doc("foo/bar", 2, map("foo", "bar")).setHasLocalMutations());
+    // Version is 0 because of remote document elision
```

Comment:
I don't think we need these comments once we use SnapshotVersion.NONE for all documents with local mutations (plus, I am not sure that the terminology used in this comment makes it obvious what is going on here)

Reply:
This IS a bit awkward, but let's decide which way to go first.

Reply:
If we keep the comment, we should phrase it differently and remove the term "elision", which requires too much knowledge to understand.

Reply:
Removed comments.